newrelic_rpm.gemspec

```diff
@@ -56,7 +56,9 @@ https://github.com/newrelic/newrelic-ruby-agent/
   s.add_development_dependency 'pry' unless ENV['CI']
   s.add_development_dependency 'bundler'
   s.add_development_dependency 'rubocop'
+  s.add_development_dependency 'rubocop-minitest'
   s.add_development_dependency 'rubocop-performance'
+  s.add_development_dependency 'rubocop-rake' if RUBY_VERSION >= '2.4.0'
```

Comment:
All checks using 2.2 failed without setting `rubocop-rake` to >= 2.4.0

Reply:
If the gem is conditionally brought in here, it will also have to be conditionally required in the YAML file.

Reply:
[Addressed here](https://github.com/newrelic/newrelic-ruby-agent/pull/1399/commits/e327d5c31ab829b4e13483dab73d8e90f485d724). Thanks for your help :)